content/about/about_people.rst

```diff
@@ -156,7 +156,8 @@ vikurtz
 .. raw:: html
 
   <br/>
-
+  
+<<<<<<< HEAD
```

Comment:
Looks like this was accidentally added. Please remove. You can then just do:
```
git commit --amend -a
git push -ff
```
Which will keep this as one commit.